spec/assessments/largestKeywordDistanceAssessmentSpec.js

```diff
@@ -0,0 +1,47 @@
+const largestKeyWordDistanceAssessment = require( "../../js/assessments/seo/largestKeywordDistanceAssessment.js" );
+const Paper = require( "../../js/values/Paper.js" );
+const Factory = require( "../helpers/factory.js" );
+const i18n = Factory.buildJed();
+
+const keywordDistanceAssessment = new largestKeyWordDistanceAssessment();
+
+describe( "An assessment to check the largest percentage of text in which no keyword occurs", function() {
+	it( "runs the largest keyword distance assessments on the paper", function() {
+		let mockPaper = new Paper( "string with the keyword", { keyword: "keyword" } );
+		let assessment = keywordDistanceAssessment.getResult( mockPaper, Factory.buildMockResearcher( 35 ), i18n );
+
+		expect( assessment.getScore() ).toEqual( 1 );
+		expect( assessment.getText() ).toEqual ( "There are some parts of your text that do not contain the keyword. Try to distribute the keyword more evenly." );
```

Comment:
Some parts of your text do not contain the keyword. Try to distribute the keyword more evenly.